unload/utils.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from os import getcwdu
+from os.path import isfile, join
+from sys import version_info
+
+from django.apps import apps
+from django.conf import settings
+
+
+def get_project_path():
+    """Get the project path"""
+
+    # Get the project path from the project's settings.py
+    if hasattr(settings, 'BASE_DIR'):
```

Comment:
try to generalize this a bit more, developers can set this variable to any name, so it would be better to find the project path on your own and based on location of settings module